mautic/mautic
# Fix input format for onesignal.

closes #6709 - OneSignal notifications with "Additional Data" are failing.

@btbounteous and the Bounteous will be providing testing


| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? | N
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #6709
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Requires OneSignal setup 
2. Create Mobile notification.
3. Under "Additional data" tab add any custom values.
4. Send push notification using one signal api and note the error

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. configure OneSignal
3. Repeat the above and there should not be an error
